Apps/Sandcastle/gallery/Custom DataSource.html

```diff
@@ -189,7 +189,7 @@
 /**
  * Asynchronously loads the GeoJSON at the provided url, replacing any existing data.
  * @param {Object} url The url to be processed.
- * @returns {Promise} a promise that will resolve when the GeoJSON is loaded.
+ * @returns {Promise.<Any>} a promise that will resolve when the GeoJSON is loaded.
```

Comment:
Here's a reference of using `Any` https://github.com/icholy/cesium/blob/bd015ae51efe0b1bf8dcfe33df8e17a9b3bdc211/Source/Core/IndexDatatype.js#L98